### PR TITLE
Minor bugfixes

### PR DESCRIPTION
**Pull request by GlancingMind**
- Fix a console error which occurs when a route is unknown to the navigation bar.
  E.g. users opens the detail page of a bug.

- When editing the title of an bug, only set it if the title is different form the old title.

### CODE REVIEW

File: webui/src/components/Header/Header.tsx [hunk 1]
```diff
@@ -67,14 +67,23 @@ const DisabledTabWithTooltip = (props: TabProps) => {
...
+        return '/code';
+      case '/pulls':
+        return '/pulls';
+      case '/settings':
+        return '/settings';
+      default:
+        // using false as value for tabs will result in no selected tab
+        return false;
+    }
+  }
```
Comment (MichaelMure) [inline webui/src/components/Header/Header.tsx:86]:
```suggestion
  function highlightTab() {
    switch (location.pathname) {
      case '/':
      case '/code':
      case '/pulls':
      case '/settings':
        return location.pathname;
      default:
        // using false as value for tabs will result in no selected tab
        return false;
    }
  }
```

What about this?

Comment (GlancingMind) [reply webui/src/components/Header/Header.tsx:86]: Makes sense. I think I could even remove the switch... Will be back soon.

Comment (GlancingMind) [reply webui/src/components/Header/Header.tsx]: This is even more [compact](https://github.com/MichaelMure/git-bug/pull/625/commits/f980d9487ed1e764a364d5a1f385a09835276962#diff-65c829cfa57d5761b963b25ffac663098388927e2ac8abc286e261c8693f1be5R74-R76) :)
What do you think?

**Comment by MichaelMure** [comment]
Merged!

**Pull request merged**